src/ntfsck.c

```diff
@@ -666,11 +666,13 @@ static int ntfsck_add_index_entry_orphaned_file(ntfs_volume *vol, s64 mft_no)
 			 * TODO: Add inode whose parent inode was deleted,
 			 *	 to root directory.
 			 */
-			ntfs_log_info("!! FOUND Deleted parent inode(%llu), inode(%llu)\n",
+			ntfs_log_verbose("!! FOUND Deleted parent inode(%llu), inode(%llu)\n",
 					(unsigned long long)MREF(parent_no),
 					(unsigned long long)ni->mft_no);
 
 delete_inodes:
+			err = 0;
+
```

Comment:
below code should be added, to prevent memory leak for parent_ni.

 676                 if (parent_ni) {
 677                         ntfs_inode_close(parent_ni);
 678                         parent_ni = NULL;
 679                 }

Reply:
Please add fixes for this.